cmd/litcli/accounts.go

```diff
@@ -26,6 +27,7 @@ var accountsCommands = []cli.Command{
 		Subcommands: []cli.Command{
 			createAccountCommand,
 			updateAccountCommand,
+			updateBalanceCommands,
```

Comment:
wondering if we cant fit this under the existing `update` category somehow?

Reply:
also, how about just having explicit credit/debit sub commands?

Reply:
> also, how about just having explicit credit/debit sub commands?

Went with this option!